From d263b3f2d8c127956a0dd1c2a12b126a9cbb6611 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Tue, 4 Dec 2012 18:50:02 -0800
Subject: [PATCH 1220/1357] MR1: CDH-9220. TestStreamingStatus failing - regression due to CDH-8955
 	  Fix TestStreamingStatus#validateTaskStatus to set finalPhase of
 	  map task to be sort
 Reason: Fix regression test failure
 Ref: CDH-9220
 Author: Karthik Kambatla

---
 .../hadoop/streaming/TestStreamingStatus.java      |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
index 1269449..dc80c9c 100644
--- a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
+++ b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestStreamingStatus.java
@@ -280,8 +280,8 @@ public class TestStreamingStatus {
       finalPhaseInTask = "reduce";
     }
     assertEquals(1, reports.length);
-    assertEquals(expectedStatus +
-        (finalPhaseInTask == null ? "" : " > " + finalPhaseInTask),
+    assertEquals(expectedStatus + " > "
+        + (finalPhaseInTask == null ? "sort" : finalPhaseInTask),
         reports[0].getState());
   }
 
-- 
1.7.0.4

